_notes/rft.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Request For Teachers
+---
+
+I believe that the RFT should be approached from the "problem" angle. 
+Meaning the teachers should be reaching for a RFT whenever they see a problem.
+Here are some examples off the top of my head:
+
+- Some of my students have a hard time coming up with compelling copywriting for their product launch
+- Some of my students don't really know how to brand themselves and their talents
+- Some of my students can't seem to find the interest in math
+- Some of my students find governance in social studies very dry
+- Some of my students find text based programming hard to understand
+- Some of my students want to learn how to use the laser printer but don't know how to
+
+
+
+------------------------------------------------------------------------------------
+# RFT Template
+This is the RFT template.
+## 1. Title
+Write a catchy title here
+e.g 
+
+## 2. Problem
+Describe what is the problem you're facing
+e.g 
+
+## 3. Context
+Give some context as to why you're facing this problem
+e.g 
```

Comment:
```
Current Context: Students currently understand fundamental of graphics and have knowledge of lighting scenes and 3d modelling.

Identified Gap: However, they lack practical insight into procedural node based programming as being applied in real-world world of animation and 3D visualisation

Ideal Outcome: Ideally, students should be able to construct a simple 3d scene with programmable elements, that sets them up in good stead for their [3d digital game art project](link to past works)
```

Reply:
## **Current Context**: Students currently understand fundamental of graphics and have knowledge of lighting scenes and 3d modelling.

## **Identified Gap**: However, they lack practical insight into procedural node based programming as being applied in real-world world of animation and 3D visualisation

## **Ideal Outcome**: Ideally, students should be able to construct a simple 3d scene with programmable elements, that sets them up in good stead for their [3d digital game art project](link to past works)